tests/TestCase.php

```diff
@@ -28,6 +28,24 @@ public function expectedException($class, $message = null)
         }
     }
 
+    protected function isPhpUnit7()
```

Comment:
this can be static

Reply:
It can be; I don't think there's a compelling reason for this to be static though?

Reply:
Normally one would write it static unless there is a compelling reason not to.